src/ZkFold/Base/Data/Vector.hs

```diff
@@ -175,3 +176,7 @@ instance (Random a, KnownNat size) => Random (Vector size a) where
 
 instance ToJSON a => ToJSON (Vector n a) where
     toJSON (Vector xs) = toJSON xs
+
+
+parFmap :: (a -> b) -> Vector size a -> Vector size b
```

Comment:
It is unclear whether this would provide any benefits (also considering conversion to list and back), so I'd advice to just use `<$>`. In case we need parallelism, using one from `Data.Vector.Strategies` is better, I guess